api/query/cache/monitor/monitor_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package monitor
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/web-platform-tests/wpt.fyi/api/query/cache/index"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+func getTestHarness(t *testing.T) (*index.MockIndex, *MockRuntime, Monitor) {
+	mockCtrl := gomock.NewController(t)
+	idx := index.NewMockIndex(mockCtrl)
+	rt := NewMockRuntime(mockCtrl)
+	mon := NewIndexMonitor(shared.NewNilLogger(), rt, time.Microsecond, 1, idx)
+	return idx, rt, mon
```

Comment:
I think you're missing a call to `mockCtrl.Finish`; you may need to return `mockCtrl` to the caller.